tests/fxl-spread_default/OPS/page_001.xhtml

```diff
@@ -0,0 +1,10 @@
+<html xmlns="http://www.w3.org/1999/xhtml" xmlns:epub="http://www.idpf.org/2007/ops" xml:lang="en">
+  <head>
+    <title>rendition:spread default</title>
+    <meta name="viewport" content="width=900, height=600" />
+    <link rel="stylesheet" href="fixed.css" />
+  </head>
+  <body id="p1">
+  	<p>Page 1. This test passes if the synthetic spread behaviour matches the reading system default or preferences.</p>
```

Comment:
I don't think this is clear enough. Can you crib from the auto test? Also, I don't see anything about preferences at https://www.w3.org/TR/epub-rs-33/#spread — is it somewhere else? I can believe that, in auto, a RS might respect user preferences (this should probably also be mentioned in the auto test) but a) I don't think (?) this is specific to the default case (applies to auto too?) and b) doesn't appear to be in the spec so maybe shouldn't be mentioned in the test?

Reply:
Also I accept, but am noting, your Canadian spelling of behaviour :)